tests/widget/modal/widget-modal.login.test.js

```diff
@@ -22,3 +22,54 @@ test('modal widget', async ({ page }) => {
   expect(await fullName.innerText()).toBe('Full name: Demo User');
   expect(await email.innerText()).toBe('Email: demo@user.com');
 });
+
+test.only('modal widget accessibly', async ({ page }) => {
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  const dialog = page.locator('dialog');
+  expect(await dialog.isVisible()).toBeFalsy();
+
+  await loginButton.click();
+  // Add just a bit of a delay to ensure dialog responds
+  await page.waitForTimeout(1000);
+  expect(await dialog.isVisible()).toBeTruthy();
+  await page.goto('widget/modal');
+  const username = page.getByRole('textbox', { name: /User Name/ });
+  const password = page.getByRole('textbox', { name: /Password/ });
+  const eye = page.getByRole('button', { name: 'show-password' });
+
+  const loginBtn = page.getByRole('button', { name: 'Sign in' });
+  await page.keyboard.press('Tab')
+  expect(username).toBeFocused();
+  await page.type(username, 'username01');
+
+  await page.keyboard.press('Tab')
+  expect(password).toBeFocused();
+  await page.type(password, 'password');
+
+  await page.keyboard.press('Tab')
+  expect(eye).toBeFocused();
+
+  await page.keyboard.press('Tab')
+  expect(loginBtn).toBeFocused();
+  await page.keyboard.press('Enter')
+
+  const error = page.getByRole('section')
```

Comment:
I'd like to make sure each element we are testing for focus is also tested for their accessible text. If I understand this correctly, the above (as an example) `page.getByRole('textbox', { name: 'User Name' });` would fail if there was no "textbox" with accessible text of "User Name", right? I'd like us to do the same for errors and subsequent journey views.